inspect.lua

```diff
@@ -28,6 +28,8 @@ local inspect ={
   ]]
 }
 
+inspect.tostring = _G.tostring
```

Comment:
I don't think we need to let this parameter be "tweakable" by end users. Please replace this line with:
``` lua
local tostring = _G.tostring
```
And revert back to plain `tostring` in the other places.

(Think about it this way: I also make use of `pairs`. But I wouldn't expose that as a "tweakable value" in `inspect.pairs`)